samples/errors5.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+)
+
+var NotEqualErr = errors.New("Inputs not equal")
+
+func main() {
+	err1 := modcheck(5, 3)
+	err2 := modcheck(5, 3)
+	fmt.Println("err1 == err2: ", err1 == err2)
+	err3 := eqcheck(1, 1)
+	err4 := eqcheck(2, 2)
+	fmt.Println("err3 == err4: ", err3 == err4)
```

Comment:
nil == nil? you should also print the errors i think

Reply:
Done.